simple-vm/ansible/roles/golang/files/go-webapp.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=go-webapp
+
+[Service]
+Type=simple
+Restart=always
+RestartSec=5s
```

Comment:
Why do we need to specify this?

Reply:
I'm assuming you're talking about the `RestartSec` field,

First of all, we added the `Restart=always` field, which in case the process `exit`s due to any reason, SystemD will restart the process without looking at the cause. In some cases, if it failed because of certain intermittent issues, restarting immediately might just cause it to fail again. So having a few seconds of delay is just a way to avoid that. The absolute value, `5` in this case, could be adjusted after observing the application and it's patterns to something like 20 or 30.